cypress/e2e/upload-download.cy.ts

```diff
@@ -0,0 +1,19 @@
+import {Upload} from "../page/index";
+
+const upload: Upload = new Upload();
+
+describe("Upload and download a file", ()=>{
+  let txt: {targetFile: string};
+  before(()=>{
+    cy.fixture("upload").then((texto)=>{
+      txt = texto;
+    });
+  });
+
+  it("Should upload a file", ()=>{
+    upload.visitTestPage();
+    upload.uploadFile(txt.targetFile);
```

Comment:
I will just pass the filename to upload, by default cypress will search the file in the fixtures folder

Reply:
It's true, please make this change

Reply:
Great!! I've already solved this one too. Thx, guys